InfiniteBlanK3T(2).CodingTracker/MainCodingTracker/DatabaseCreation.cs

```diff
@@ -0,0 +1,95 @@
+﻿using Microsoft.Data.Sqlite;
+using System.Configuration;
+
+namespace CodingTracker
+{    
+    public class DatabaseCreation
+    {
+        Validation val = new();
```

Comment:
Looks like this isn't being used? 

![image](https://user-images.githubusercontent.com/108711476/218888253-5dc03a2d-4b0e-4acc-96aa-f7c0efd285d5.png)